From c61c66fffccdf71993c3cc7daaf802cdea9331f3 Mon Sep 17 00:00:00 2001
From: Yi Zhao <yi.zhao@windriver.com>
Date: Mon, 3 Aug 2020 10:51:49 +0800
Subject: [PATCH] policy/modules/system/mount: allow mount command to mount any
 directory or file

Fixes:
avc:  denied  { read } for  pid=306 comm="mount" name="fstab" dev="sdb3"
ino=42 scontext=system_u:system_r:mount_t:s0-s15:c0.c1023
tcontext=system_u:object_r:unlabeled_t:s0 tclass=file permissive=0

Upstream-Status: Inappropriate [ostree specific]

Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 policy/modules/system/mount.te | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/policy/modules/system/mount.te b/policy/modules/system/mount.te
index 1fbd467b4..eb8a9aeec 100644
--- a/policy/modules/system/mount.te
+++ b/policy/modules/system/mount.te
@@ -10,7 +10,7 @@ policy_module(mount, 1.24.0)
 ## Allow the mount command to mount any directory or file.
 ## </p>
 ## </desc>
-gen_tunable(allow_mount_anyfile, false)
+gen_tunable(allow_mount_anyfile, true)
 
 attribute_role mount_roles;
 roleattribute system_r mount_roles;
-- 
2.17.1

